soh/soh/SohMenuBar.cpp

```diff
@@ -253,6 +255,21 @@ void DrawSettingsMenu() {
                 LUS::Context::GetInstance()->GetWindow()->SetResolutionMultiplier(CVarGetFloat("gInternalResolution", 1));
             };
             UIWidgets::Tooltip("Multiplies your output resolution by the value inputted, as a more intensive but effective form of anti-aliasing");
+           
+            // Tina's resolution thingy
```

Comment:
I'm not sure if this comment is really needed since it doesn't seem to have a clear purpose. Imagine if everyone started adding their usernames to every section they contributed – it might get a bit cluttered

Reply:
Yeah I already removed that right after making the PR but hadn't pushed the change yet, don't worry. Will Resolve with the next commit.